DiamondLightSource/SynchWeb
# [WIP] Setup: Initialize cypress test for frontend app

#### What does this PR do?
This PR initial setup for running cypress end-to-end for the synchweb application. It also contains some test that have been written to ensure that everything works.

#### How can it be tested?
- On your terminal, set the environment variable of the base_url like so: `CYPRESS_BASE_URL=<the_url_running_synchweb>`
- Copy the `.env.sample` file to a `.env` file and fill in its correct values
- Run `npm run cypress:open` to see the test run on the browser or `npm run cypress:run` to see the test run in headless mode.
